DEPLOYNOTES.rst

```diff
@@ -3,10 +3,19 @@
 Deploy and Upgrade notes
 ========================
 
+1.0
+---
+
+XML sitemaps require update to Solr schema to track last modification
+time of indexed items. Update the schema and reindex everything.
+
+  python manage.py solr_schema
+  python manage.py index
```

Comment:
are we no longer recommending doing the `python manage.py index -c all -i none` thing prior to indexing? can't remember if it's good practice or just habit

Reply:
I don't think we need it in this case because we're just adding a new auto-generated field (although it doesn't hurt to run it).

Reply:
makes sense